src/net/server/channel/handlers/DueyHandler.java

```diff
@@ -316,8 +316,9 @@ private void addItemToDB(Item item, int quantity, int mesos, String sName, int r
         Connection con = null;
         try {
             con = DatabaseConnection.getConnection();
-            try (PreparedStatement ps = con.prepareStatement("SELECT * FROM dueypackages dp LEFT JOIN dueyitems di ON dp.PackageId=di.PackageId WHERE RecieverId = ?")) {
+            try (PreparedStatement ps = con.prepareStatement("SELECT * FROM dueypackages dp LEFT JOIN dueyitems di ON dp.PackageId=di.PackageId WHERE ReceiverId = ? OR ReceiverAccountId = ?")) {
```

Comment:
can we avoid using `SELECT *` and pull only the relevant columns? We can easily save ~10 - 20% of its current bandwidth by doing so.

Reply:
Unless you want to do individual queries for the package items based on package type, the best I can do is reduce result size by 3 columns, which is what I did in the latest commit. Dueyitems.* is unavoidable because equipment reconstruction requires all the data in the record.